python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -120,7 +145,11 @@ def __init__(
             raise DagsterDbtProjectNotFoundError(f"project_dir {project_dir} does not exist.")
 
         packaged_project_dir = Path(packaged_project_dir) if packaged_project_dir else None
-        if not using_dagster_dev() and packaged_project_dir and packaged_project_dir.exists():
+        if (
+            not manifest_preparer.using_dagster_dev()
+            and packaged_project_dir
+            and packaged_project_dir.exists()
+        ):
```

Comment:
my understanding was that this symlink approach is trading [1] for being able to remove this conditional